impeller/renderer/backend/gles/capabilities_gles.h

```diff
@@ -15,7 +15,7 @@ namespace impeller {
 class ProcTableGLES;
 
 struct CapabilitiesGLES {
```

Comment:
It's really odd that this doesn't extend Capabilities like CapabilitiesVK and we have to pipe these in later. I guess that base class came after the OpenGL ES prototype. Something to tackle later.

Reply:
Done